Common/UnitDefinitions/PressureChangeRate.json

```diff
@@ -57,6 +57,44 @@
           "Abbreviations": [ "атм/с" ]
         }
       ]
+    },
+	    {
+      "SingularName": "PoundForcePerSquareInchPerSecond",
+      "PluralName": "PoundsForcePerSquareInchPerSecond",
+      "FromUnitToBaseFunc": "x*6.894757293168361e3",
+      "FromBaseToUnitFunc": "x/6.894757293168361e3",
+      "Prefixes": [ "Kilo", "Mega" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "psi/s", "lb/in²/s" ],
+          "AbbreviationsForPrefixes": { "Kilo": ["kipf/in²/s", "ksi/s" ]}
```

Comment:
Is ksi/s more widely used than kipf/in2/s? If so, consider swapping the order of these. Seems more consistent with psi/s being the first choice, too.

The first abbreviation is the one used by ToString() when no custom formatting is specified: https://github.com/angularsen/UnitsNet/wiki/String-Formatting

Reply:
I think you're right (that's what I think too), but actually i just copied the abbreviations from PoundForcePerSquareInch in Pressure.json

To keep it logical, i think we should change PoundForcePerSquareInch definition too, but as you said, the first abbreviation is used by ToString(), so it could introduce some breaking (even if i never see "kipf/in²" in any standard/document).

What do you think i should do?

Reply:
I think we can risk a minor breaking change and swap the order of Pressure too, for the sake of consistency.